packages/sdk/src/types/ContractAddress.ts

```diff
@@ -21,6 +23,10 @@ class ContractAddress {
         /** The subindex of the smart contract address. */
         public readonly subindex: bigint
     ) {}
+
+    public toJSON(): ContractAddressLike<bigint> {
+        return { index: this.index, subindex: this.subindex };
+    }
```

Comment:
It's a bit scuffed that toJSON is not compatible with stringify, but I guess there's no way around it... :smiling_face_with_tear:

Reply:
Yeah, well it doesn't work with the browser wallet anyway, so probably have to scrap this anyway.